lib/CL/clCreateSubDevices.c

```diff
@@ -185,6 +185,9 @@ POname(clCreateSubDevices)(cl_device_id in_device,
          new_devs[i]->core_start = sum;
        sum += new_devs[i]->core_count;
      }
+     for (i = 0; i < count_devices; ++i) {
```

Comment:
Can you run `tools/scripts/format-diff.sh` or `format-branch.sh` to fix the style (GNU C style in C files)?

Reply:
Will do, thanks for the review.